src/sdgio-edit.owl

```diff
@@ -158,6 +164,61 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020236>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020237>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020300>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020308>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020309>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020310>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020311>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020312>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020313>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020317>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020318>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020319>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020320>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020325>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020326>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020327>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020328>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020329>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020331>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020334>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020335>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020338>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020339>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020341>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020342>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020343>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020349>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020359>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020360>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020361>))
```

Comment:
- change spelling to British English 
- moved SDGIO_00020362 to capacity development following its definition and linking it through parthood to 'planned process to apply organisational policies' under management processes